<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KickDialog</class>
 <widget class="QDialog" name="KickDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>254</width>
    <height>190</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Kick players</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#KickDialog {
	background:url(:/images/background_connection.png);
	width:200%;
}
#KickDialog QLabel{
	color:white;
}

#KickDialog QPushButton {
background:url(:/images/btn.png) transparent no-repeat;
border:0px;
width:80px;
height:20px;
}
#KickDialog QPushButton:hover {
background:url(:/images/btn_hover.png) transparent no-repeat;
border:0;
width:70px;
height:25px;
}
#KickDialog QPushButton:pressed {
background:url(:/images/btn_hover.png) transparent no-repeat;
border:0;
width:70px;
height:25px;
}
#KickDialog QListWidget{
background-color:#4a4a4a;
color:white;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Select the players to be kicked:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QListWidget" name="listWidget"/>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="app.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>KickDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>KickDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
